src/main/resources/application.yml

```diff
@@ -0,0 +1,10 @@
+logging:
+  level:
+    root: info
+spring:
+  application:
+    name: deckcompare
+  main:
+    banner-mode: console
```

Comment:
console is the default value. See https://docs.spring.io/spring-boot/docs/current/reference/html/common-application-properties.html